src/main/java/com/google/step/datamanager/RestaurantManagerDatastore.java

```diff
@@ -124,9 +127,10 @@ public void deleteRestaurant(long id) {
   private Restaurant transformEntityToRestaurant(Entity restaurantEntity) {
     String name = (String) restaurantEntity.getProperty("name");
     String photoUrl = (String) restaurantEntity.getProperty("photoUrl");
+    long posterId = (long) restaurantEntity.getProperty("posterId");
```

Comment:
if posterId doesn't exist (like past restaurant entities already in database), will this cause any errors? If yes, then might be better to maybe assign posterId to be -1, or something else, like the admin's Id

Reply:
I add a admin user using a service account email in the restaurant generator.

Reply:
yea i mean currently there are some restaurants in the live datastore that doesn't have a posterId. Might need to remember to nuke everything after you merge, or manually add in posterIds to the ones without posterIds.

Reply:
ok I see